hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreEngine.java

```diff
@@ -202,7 +205,7 @@ protected final void createComponentsOnce(Configuration conf, HStore store,
     this.openStoreFileThreadPoolCreator = store.getHRegion()::getStoreFileOpenAndCloseThreadPool;
     this.storeFileTracker = createStoreFileTracker(conf, store);
     assert compactor != null && compactionPolicy != null && storeFileManager != null
-      && storeFlusher != null && storeFileTracker != null;
```

Comment:
Static analysis says `storeFileTracker` is always null.

Reply:
never null?

Reply:
That was the nit that IntelliJ provided, yes. Reading down through `createStoreFileTracker()`, it never returns null. If it fails, it throws.